src/test/TxTests.cpp

```diff
@@ -186,7 +186,12 @@ applyCheck(TransactionFramePtr tx, Application& app, bool checkSeqNum)
 
             // verify that the fee got processed
             auto ltxDelta = ltxFeeProc.getDelta();
-            REQUIRE(ltxDelta.entry.size() == 1);
+            // the delta has a MAX_SEQ_NUM_TO_APPLY entry starting from v19
+            auto size =
+                protocolVersionStartsFrom(ledgerVersion, ProtocolVersion::V_19)
+                    ? 2
+                    : 1;
+            REQUIRE(ltxDelta.entry.size() == size);
```

Comment:
why do we return things other than `LEDGER_ENTRY` when calling `getDelta` ? I can't find any use for this

Reply:
It was used to check for dangling sponsorship entries, but we changed that here - https://github.com/stellar/stellar-core/commit/f59f047ac3f99d88b7a637a10b6e35a412de7297. We can change it to only return `LEDGER_ENTRY` now.